benchmarks/yadmark/data/observables.py

```diff
@@ -31,6 +31,10 @@
         "observable_names": ["F2light", "F2total", "FLtotal", "F3total"],
         "kinematics": default_kinematics,
     },
+    2: {
```

Comment:
See the other comment, simply split this config out of this dictionary. Or in this case maybe it's better to simply update the `default_config[1]`, since the two are so close to be almost the same.